blocks/init/src/blocks/components/hamburger/hamburger-style.scss

```diff
@@ -0,0 +1,78 @@
+/* stylelint-disable max-nesting-depth */
+
+$hamburger: (
+  colors: (
+    line: #2C2C2C,
+  ),
+  width: 24px,
```

Comment:
if you are using this only in one place then there is no need for this to go to the map

Reply:
Since this is hamburger menu toggle I would place all major variables in the map. 
Even separate them in map specific for lines and a map specific to the wrapper of the hamburger lines.
This way it is easier to change line width, height and overall size of the hamburger.

Reply:
I'm with @gabriel-glo on this one, I've actually extracted more values in the map and added comments explaining each line - it's much easier to customise the hamburger this way without having add any style.

Reply:
sure awesome keep it like this then